ignite/metrics/__init__.py

```diff
@@ -8,7 +8,15 @@
 from ignite.metrics.mean_absolute_error import MeanAbsoluteError
 from ignite.metrics.mean_pairwise_distance import MeanPairwiseDistance
 from ignite.metrics.mean_squared_error import MeanSquaredError
-from ignite.metrics.metric import BatchFiltered, BatchWise, EpochWise, Metric, MetricUsage
+from ignite.metrics.metric import (
+    BatchFiltered,
+    BatchWise,
+    EpochWise,
+    Metric,
+    MetricUsage,
+    reinit__is_reduced,
+    sync_all_reduce,
```

Comment:
If you expose these methods as `ignite.metrics.reinit__is_reduced/sync_all_reduce`, can you also check locally the docs and the links on them here : https://pytorch.org/ignite/metrics.html#ignite.metrics.metric.sync_all_reduce
and maybe add links to those methods here : https://pytorch.org/ignite/metrics.html#metrics-and-distributed-computations
currently they are not clickable...

Reply:
I did it due to pycharm but maybe it is not necessary to expose.

Reply:
OK, let's keep them where they are. And I create an issue to fix unclickable link on them.

Reply:
I mean, let's not expose those methods